js/src/schemas.ts

```diff
@@ -587,3 +593,5 @@ export type UsageMetadata = {
    */
   output_token_details?: OutputTokenDetails;
 };
+
+export type Attachments = Record<string, [string, Uint8Array]>;
```

Comment:
_question_: do we want to support ArrayBuffer as well? Might save a unnecessary array copy

Reply:
Oh - I think that makes sense? Will have to look into it

Reply:
Added!